tcms/rpc/tests/test_auth.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init
+
+from xmlrpc.client import Fault as XmlRPCFault
+from django.contrib.sessions.models import Session
+from tcms.rpc.api.user import User
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests import UserFactory
+
+
+class TestAuthLogin(APITestCase):
+    """Test Auth.login method"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        #
+        self.rpc_client.Auth.logout()
+        self.tester = UserFactory()
+        self.password = "password"
+        self.tester.set_password(self.password)
+        self.tester.save()
+
+    def test_login_with_username(self):
+        self.rpc_client.Auth.logout()
+        sesson_id = self.rpc_client.Auth.login(self.tester.username, self.password)
+        session = Session.objects.get(session_key=sesson_id).get_decoded()
+
+        self.assertIsNotNone(sesson_id)
+        self.assertEqual(User.objects.get(pk=session.get("_auth_user_id")), self.tester)
```

Comment:
this isn't necessary b/c effectively yu are checking whether or not Django's authentication backend recorded this user into the session or something like that. This is a concern of Django's own test suite to validate. If we get a session_id we are good to go.